docker-compose.yml

```diff
@@ -9,6 +9,10 @@ services:
     depends_on:
       - mongo
       - elastic
+    environment:
+      cors.allowed.origins: http://localhost:3000
```

Comment:
Sure, can do. 

Though I'll note having tried this that it appears the OlogIT etc tests don't appear to run successfully regardless and I don't see them running in the CI/CD here? 

So that at least I think should be addressed separately. But at least given how the tests are run, using the same args in the integration tests seems reasonable. 

Except cors allowed origins 🤦 
I realized upon inspection that it already defaults to localhost:3000 in the java configuration files when the value is unset.